### ISSUE TITLE
item.bounds.select handles fillColor issue?

### ISSUE THREAD

**Issue by FeaRCODE** [paperjs/paper.js#1019]
Should ALL the handles inherit the color passed to `obj.selectedColor`? As pictured below, this is the undesired result (they should all have a fillColor). 
```
paper.settings.handleSize = 8;
obj.selectedColor = "#COLOR";
obj.bounds.selected = true;
```

![image](https://cloud.githubusercontent.com/assets/6901336/13934062/1ada801c-ef7e-11e5-9094-b922eef5a1d8.png)

**Comment by FeaRCODE** [comment]
After looking into it further, I also noticed the bounds are selected as well as the bounds of the path. IS there a way to ONLY select the bounds rectangle?

**Comment by lehni** [comment]
The white fill means that the path is selected but the segments aren't. The fact that the path automatically gets selected when you select the bounds was a bug that was addressed in #769. Please try again with the latest dev build, it should be fixed.

**Comment by FeaRCODE** [comment]
@lehni My bad! Feel free to close this if you feel it's been completely resolved. Thanks!
